### ISSUE TITLE
Add github actions maven deployment

### ISSUE THREAD

**Issue by tischi** [tischi/imaris-writer#6]

**Comment by tischi** [comment]
@imagejan @K-Meech is this necessary in the pom?

```
		<!-- NB: Deploy releases to the SciJava Maven repository. -->
		<releaseProfiles>sign,deploy-to-scijava</releaseProfiles>
```

**Comment by imagejan** [comment]
@tischi yes, it's required to deploy to `maven.scijava.org` (instead of Maven Central).
These lines are added automatically by the `github-actionify.sh` script.

**Comment by tischi** [comment]
> These lines are added automatically by the github-actionify.sh script.

Ah, good point! I was now manually copying the `.github` folder and changing the pom. 
In theory, running `github-actionify.sh` should do all this?

**Comment by imagejan** [comment]
> running github-actionify.sh should do all this?

Yes 🙂

**Comment by tischi** [comment]
@ctrueden @imagejan could you please double-check whether deploying to maven should work for this repo and organisation?

**Comment by tischi** [comment]
@ctrueden I made you an _owner_ of this organization in case the secrets still need to be added.

**Comment by ctrueden** [comment]
@tischi The secrets were already added to bigdataprocessor org on Nov 11, 2021.

The reason your release didn't deploy is due to a typo in the repository path in your POM. From [the build log](https://github.com/bigdataprocessor/imaris-writer/actions/runs/3113615793/jobs/5048469757#step:5:23):
```
No deploy -- repository fork: https://github.com/bigdataprocessor/imaris-writer != https://github.com/bigdataprocesser/imaris-writer
```
I fixed it (f566e824d6a88eddec010488ec9abffd1d02571c), and the 1.0.6-SNAPSHOT is [deployed now](https://maven.scijava.org/#nexus-search;gav~de.embl.cba~imaris-writer~1.0.6-SNAPSHOT~~). You should be able to cut a release of 1.0.7 any time.
